### PR TITLE
Add SANS direct beam script as more recent demo

### PR DESCRIPTION
**Pull request by SimonHeybrock**
This is meant to provide a more recent overview and demo of scipp's features and power. Actual scripts used are not part of the main scipp repo.

### CODE REVIEW

File: scipp-jupyter-demo.Dockerfile [hunk 1]
```diff
@@ -42,6 +52,16 @@ RUN conda install --yes \
 # Avoid weird tornado AttributeError
 RUN pip install --upgrade nbconvert
 
+# Get code for SANS direct-beam iteration demo
+RUN OWD=$(pwd) && \
```
Comment (DanNixon) [inline scipp-jupyter-demo.Dockerfile:56]:
```suggestion
RUN PWD=$(pwd) && \
```

Comment (SimonHeybrock) [reply scipp-jupyter-demo.Dockerfile:56]: Doesn't `PWD` get overwritten when I change directory?

Comment (DanNixon) [reply scipp-jupyter-demo.Dockerfile]: Ah, yes. My point was more that it was a bad name.

Comment (SimonHeybrock) [reply scipp-jupyter-demo.Dockerfile]: I figured that it was unnecessary anyway, removed now.

**Comment by DanNixon** [comment]
As the main thing here is a workflow and not really anything directly scipp related I think we should create images in `scipp/ess` for these. This is a better from an organisational stand and will scale better as we have more workflows.

**Comment by SimonHeybrock** [comment]
> As the main thing here is a workflow and not really anything directly scipp related I think we should create images in `scipp/ess` for these. This is a better from an organisational stand and will scale better as we have more workflows.

We should probably planning on removing the entire docker stuff from this repo? Even what we have now does not really belong here?

**Comment by SimonHeybrock** [comment]
@DanNixon I opened #1220 so we can start working on solution-finding.

**Pull request merged**